frontend/src/scenes/plugins/edit/PluginDrawer.tsx

```diff
@@ -172,6 +174,33 @@ export function PluginDrawer(): JSX.Element {
                                 </div>
                             ) : null}
 
+                            {user?.organization?.plugins_access_level === PluginsAccessLevel.Root && (
+                                <>
+                                    <h3 className="l3" style={{ marginTop: 32 }}>
+                                        Installation
+                                    </h3>
+                                    <Tooltip
+                                        title={
+                                            <>
+                                                Enabling this will mark this plugin as installed for{' '}
+                                                <b>all organizations</b> in this instance of PostHog.
+                                            </>
+                                        }
+                                        placement="bottom"
+                                    >
+                                        <Checkbox
+                                            checked={editingPlugin.is_global}
+                                            onChange={(e) =>
+                                                patchPlugin(editingPlugin.id, {
+                                                    is_global: e.target.checked,
+                                                })
+                                            }
+                                        />
+                                        <span style={{ paddingLeft: 10 }}>Mark as global</span>
```

Comment:
Shouldn't this be inside the `<Checkbox>` tag? It's in any case not clickable.

I also find it a bit confusing to have a checkbox that autosaves the form, though all things considered, it's probably a fine compromise.

![image](https://user-images.githubusercontent.com/53387/110852288-785bbe00-82b2-11eb-8cb7-1966d83e9d44.png)

Reply:
Yep, should be clickable.

This does bypass "Save" which is not ideal UX, but unfortunately it has to, as this is a control for the Plugin model, while the rest of the drawer is a control for PluginConfig. Different endpoints.
I actually wanted to make this a button next to "Uninstall" (which does correspond to the Plugin model), but unfortunately there's not enough space in the footer.